Cartfile.private

```diff
@@ -1,2 +1,2 @@
-github "thoughtbot/Curry"
+github "thoughtbot/Curry" "tvos"
 github "thoughtbot/Runes" >= 3.0.0
```

Comment:
But the runes code is directly added to the Argo project so we don't actually use any Runes targets